### ISSUE TITLE
Add aria labels to Integrity and Compliance Helpline number on Helpful VA phone numbers page

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#40443]
## Issue Description

As a style guide rule, all phone numbers on VA.gov need to use aria labels. This is for accessibility so that screen readers recite the number correctly. 

Currently, the Integrity and Compliance Helpline phone number toward the bottom of our [Helpful VA phone numbers page](https://www.va.gov/resources/helpful-va-phone-numbers/) is missing the aria labels. We need to [add the source code](https://design.va.gov/content-style-guide/dates-and-numbers#phone-numbers) to each number in the Drupal page. 

## Tasks
- [x] Add aria labels to the helpline in the Drupal node (Aliyah)
- [x] Confirm aria labels were added correctly (Randi)
- [x] Publish page (Randi)

## Acceptance Criteria
- [x] All numbers on live page have aria labels

**Comment by RLHecht** [comment]
Thanks @aliyahblackmore I published the page. Moving to validate and I'll close once the page is live.

**Comment by RLHecht** [comment]
Changes are live. Closing ticket.
